h2o-docs/src/product/training-models.rst

```diff
@@ -0,0 +1,662 @@
+                              x = predictors, 
+                              training_frame = train, 
+                              validation_frame = valid)
+
+    # Eval performance:
+    perf <- h2o.performance(iris_kmeans)
+    perf
+
+    H2OClusteringMetrics: kmeans
+    ** Reported on training data. **
+
+    Total Within SS:  63.09516
+    Between SS:  483.8141
+    Total SS:  546.9092 
+    Centroid Statistics: 
+      centroid     size within_cluster_sum_of_squares
+    1        1 36.00000                      11.08750
+    2        2 51.00000                      30.78627
+    3        3 36.00000                      21.22139
+
+
+   .. code-tab:: python
+
+    import h2o
+    from h2o.estimators import H2OKMeansEstimator
+    h2o.init()
+
+    # Import the iris dataset into H2O:
+    iris = h2o.import_file("http://h2o-public-test-data.s3.amazonaws.com/smalldata/iris/iris_wheader.csv")
+
+    # Set the predictors:
+    predictors = ["sepal_len", "sepal_wid", "petal_len", "petal_wid"]
+
+    # Split the dataset into a train and valid set:
+    train, valid = iris.split_frame(ratios=[.8], seed=1234)
+
+    # Build and train the model:
+    iris_kmeans = H2OKMeansEstimator(k=10, 
+                                     estimate_k=True, 
+                                     standardize=False, 
+                                     seed=1234)
+    iris_kmeans.train(x=predictors, 
+                      training_frame=train, 
+                      validation_frame=valid)
+
+    # Eval performance:
+    perf = iris_kmeans.model_performance()
+    perf
+
+    ModelMetricsClustering: kmeans
+    ** Reported on train data. **
+    
+    MSE: NaN
+    RMSE: NaN
+    Total Within Cluster Sum of Square Error: 63.09516069071749
+    Total Sum of Square Error to Grand Mean: 546.9092331233204
+    Between Cluster Sum of Square Error: 483.8140724326029
+
+    Centroid Statistics: 
+        centroid    size    within_cluster_sum_of_squares
+    --  ----------  ------  -------------------------------
+        1           36      11.0875
+        2           51      30.7863
+        3           36      21.2214
+
+Anomaly Detection Example
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This example uses the :ref:`isoforest` algorithm to detect anomalies in the Electrocardiograms (ECG) dataset.
+
+.. tabs::
+   .. code-tab:: r R
+
+	library(h2o)
+	h2o.init()
+
+	# import the ecg discord datasets:
+	train <- h2o.importFile("http://s3.amazonaws.com/h2o-public-test-data/smalldata/anomaly/ecg_discord_train.csv")
+	test <- h2o.importFile("http://s3.amazonaws.com/h2o-public-test-data/smalldata/anomaly/ecg_discord_test.csv")
+
+	# train using the `sample_size` parameter:
+	isofor_model <- h2o.isolationForest(training_frame = train, 
+	                                    sample_size = 5, 
+	                                    ntrees = 7, 
+	                                    seed = 12345,
+	                                    validation_frame = test)
```

Comment:
Get this error when running this line: "Error in h2o.isolationForest(training_frame = train, sample_size = 5,  : 
  unused argument (validation_frame = test)"